### DISCUSSION TITLE
Own score panel in multiplayer result screen is missing placement rank when not expanded

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#15983] [Q&A] [answered]
You can see that # 2 on my panel is missing when it's not the expanded one. I've kind of always known this but today I felt like posting about it

![Screenshot_26](https://user-images.githubusercontent.com/53081750/145071323-1a05ca0d-a5da-45b1-96cd-f57f68eaeaaa.png)
![Screenshot_27](https://user-images.githubusercontent.com/53081750/145071320-4759bde3-a303-4c2b-a30e-127a56e5e661.png)

**Comment by bdach** [comment, marked as answer]
#15985
